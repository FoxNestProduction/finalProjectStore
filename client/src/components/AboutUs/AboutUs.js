/* eslint-disable max-len */
import React, { memo } from 'react';
import { Container, Typography, Grid, Paper } from '@mui/material';

const AboutUs = () => {
  return (
    <Container sx={{ paddingBottom: '30px', color: 'text.primary' }}>
      <Typography variant="h2" component="h2" sx={{ marginTop: '30px', paddingLeft: '10px' }} gutterBottom>
        About Us
      </Typography>
      <Grid container spacing={3}>
        <Grid item xs={12}>
          <Paper elevation={3} sx={{ padding: '20px' }}>
            <Typography variant="h3" gutterBottom>
              Our Story
            </Typography>
            <Typography>
              Welcome to Eatly, your destination for delicious dining experiences! We are passionate about connecting food lovers
              with the best restaurants in town. Our journey began with a simple idea: to make it easier for people to discover
              and enjoy great food. We believe that every meal is an opportunity to create wonderful memories, and we&apos;re here
              to make those moments special.
            </Typography>
          </Paper>
        </Grid>
        <Grid item xs={12}>
          <Paper elevation={3} sx={{ padding: '20px' }}>
            <Typography variant="h3" component="h3" gutterBottom>
              Our Mission
            </Typography>
            <Typography>
              At Eatly, our mission is to provide a seamless and delightful dining experience. We work tirelessly to bring you a
              diverse selection of restaurants, each with its own unique flavors and culinary traditions. Whether you&apos;re
              craving comfort food, international cuisine, or something in between, we&apos;ve got you covered. We&apos;re
              committed to helping you explore new tastes and create unforgettable food adventures.
            </Typography>
          </Paper>
        </Grid>
        <Grid item xs={12}>
          <Paper elevation={3} sx={{ padding: '20px' }}>
            <Typography variant="h3" component="h3" gutterBottom>
              Our Team
            </Typography>
            <Typography>
              Our dedicated team of food enthusiasts and tech experts is at the heart of everything we do. We&apos;re passionate
              about curating the best dining options for you, ensuring that every restaurant we partner with meets our high
              standards of quality and service. Your satisfaction is our top priority, and we&apos;re here to assist you every
              step of the way.
            </Typography>
          </Paper>
        </Grid>
        <Grid item xs={12}>
          <Paper elevation={3} sx={{ padding: '20px' }}>
            <Typography variant="h3" component="h3" gutterBottom>
              Contact Us
            </Typography>
            <Typography>
              If you have any questions, feedback, or inquiries, don&apos;t hesitate to reach out to us. We&apos;d love to hear
              from you and assist with any requests. You can contact us at
              <a href="mailto:info@eatly.com" style={{ fontWeight: 'bold', color: '#6C5FBC' }}>
                {' '}
                info@eatly.com
                {' '}
              </a>
              or through our
              <a href="/contact" style={{ fontWeight: 'bold', color: '#6C5FBC' }}>
                {' '}
                customer support page
                {' '}
              </a>
              .
            </Typography>
          </Paper>
        </Grid>
      </Grid>
    </Container>
  );
};

export default memo(AboutUs);
