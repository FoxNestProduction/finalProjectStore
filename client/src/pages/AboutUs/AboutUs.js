import React from 'react';

const AboutUsPage = () => {
  return (
    <h1>AboutUs</h1>
  );
};

export default AboutUsPage;
