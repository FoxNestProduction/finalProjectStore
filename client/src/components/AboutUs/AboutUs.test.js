import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router';
import AboutUs from './AboutUs';

const Component = (props) => {
  return (
    <MemoryRouter>
      <AboutUs />
    </MemoryRouter>
  );
};

describe('Snapshot test', () => {
  test('should render AboutUs', () => {
    const { asFragment } = render(<Component />);

    expect(asFragment()).toMatchSnapshot();
  });
});
