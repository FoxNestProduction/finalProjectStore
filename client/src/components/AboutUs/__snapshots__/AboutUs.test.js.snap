// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test should render AboutUs 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-9tnhgm-MuiContainer-root"
  >
    <h2
      class="MuiTypography-root MuiTypography-h2 MuiTypography-gutterBottom css-1aebo1m-MuiTypography-root"
    >
      About Us
    </h2>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-zow5z4-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation3 css-10u5m3t-MuiPaper-root"
        >
          <h3
            class="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom css-1mktp8m-MuiTypography-root"
          >
            Our Story
          </h3>
          <p
            class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
          >
            Welcome to Eatly, your destination for delicious dining experiences! We are passionate about connecting food lovers with the best restaurants in town. Our journey began with a simple idea: to make it easier for people to discover and enjoy great food. We believe that every meal is an opportunity to create wonderful memories, and we're here to make those moments special.
          </p>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation3 css-10u5m3t-MuiPaper-root"
        >
          <h3
            class="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom css-1mktp8m-MuiTypography-root"
          >
            Our Mission
          </h3>
          <p
            class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
          >
            At Eatly, our mission is to provide a seamless and delightful dining experience. We work tirelessly to bring you a diverse selection of restaurants, each with its own unique flavors and culinary traditions. Whether you're craving comfort food, international cuisine, or something in between, we've got you covered. We're committed to helping you explore new tastes and create unforgettable food adventures.
          </p>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation3 css-10u5m3t-MuiPaper-root"
        >
          <h3
            class="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom css-1mktp8m-MuiTypography-root"
          >
            Our Team
          </h3>
          <p
            class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
          >
            Our dedicated team of food enthusiasts and tech experts is at the heart of everything we do. We're passionate about curating the best dining options for you, ensuring that every restaurant we partner with meets our high standards of quality and service. Your satisfaction is our top priority, and we're here to assist you every step of the way.
          </p>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation3 css-10u5m3t-MuiPaper-root"
        >
          <h3
            class="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom css-1mktp8m-MuiTypography-root"
          >
            Contact Us
          </h3>
          <p
            class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
          >
            If you have any questions, feedback, or inquiries, don't hesitate to reach out to us. We'd love to hear from you and assist with any requests. You can contact us at
            <a
              href="mailto:eatly.supp@gmail.com"
              style="font-weight: bold; color: rgb(108, 95, 188);"
            >
               eatly.supp@gmail.com 
            </a>
            or through our
            <a
              href="/contact"
              style="font-weight: bold; color: rgb(108, 95, 188);"
            >
               customer support page 
            </a>
            .
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
